crates/astria-composer/local.env.example

```diff
@@ -10,6 +10,17 @@ ASTRIA_COMPOSER_NO_OTEL=false
 # If false span data is written to stdout only if it is connected to a tty.
 ASTRIA_COMPOSER_FORCE_STDOUT=false
 
+# If true uses an exceedingly pretty human readable format to write to stdout.
+# If false uses JSON formatted OTEL traces.
+# This does nothing unless stdout is connected to a tty or
+# `ASTRIA_COMPOSER_FORCE_STDOUT` is set to `true`.
+ASTRIA_COMPOSER_PRETTY_PRINT=false
+
+# If set to any non-empty value removes ANSI escape characters from the pretty
+# printed output. Note that this does nothing unless `ASTRIA_COMPOSER_PRETTY_PRINT`
+# is set to `true`.
+NO_COLOR=
```

Comment:
should this be prefixed with `ASTRIA_...`?

Reply:
This comes from the tracing-subscriber crate and follows https://no-color.org/